internal/eval/eval.go

```diff
@@ -1638,7 +1638,13 @@ func evalCommandCount() []byte {
 	return clientio.Encode(diceCommandsCount, false)
 }
 
+// evalCommandGetKeys helps identify which arguments in a redis command
+// are interpreted as keys.
+// This is useful in analying long commands / scripts
 func evalCommandGetKeys(args []string) []byte {
+	if len(args) == 0 {
+		return diceerrors.NewErrArity("COMMAND GETKEYS")
```

Comment:
Please update this to follow redis format
```
127.0.0.1:6379> COMMAND GETKEYS
(error) ERR wrong number of arguments for 'command|getkeys' command
```

Reply:
@lucifercr07 - fixed.